rabbitmq-messaging-topology-operator.yaml

```diff
@@ -49,6 +49,17 @@ subpackages:
           mkdir -p ${{targets.subpkgdir}}/
           ln -sf /usr/bin/manager ${{targets.subpkgdir}}/manager
 
+  - name: ${{package.name}}-bitnami-compat
+    dependencies:
+      runtime:
+        - ${{package.name}}
+        - ${{package.name}}-compat
+    description: "compat package for ${{package.name}} image"
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/opt/bitnami/rmq-messaging-topology-operator/bin/manager
+          ln -sf /usr/bin/manager ${{targets.contextdir}}/opt/bitnami/rmq-messaging-topology-operator/bin/manager
+
```

Comment:
Can we have a `readlink` test here? And a `runs:` pipeline for simple `--help` test for the created symlink?

Reply:
Yes!